python/GafferImageUI/CatalogueUI.py

```diff
@@ -93,7 +93,7 @@ def title( self ) :
 		return self.__title.value
 
 	## Calls `_imageCellData()`.
-	def cellData( self, path, canceller ) :
+	def cellData( self, path, canceller = None ) :
```

Comment:
Would be good to apply this same fix to `SleepingColumn` in `PathListingWidgetTest`, so the unit tests continue to provide best-practice examples.

Reply:
Fixed in 4fbc82adfc23be6e3b56a12fb0c03a638a330552